### ISSUE TITLE
Unable to set logos for custom systems

### ISSUE THREAD

**Issue by trashcatt** [anthonycaccese/art-book-next-es-de#23]
Hello, for whatever reason I am unable to set SVG logos for custom systems. I made a couple of systems for Pokemon Rom Hacks and the backgrounds took just fine but the SVGs are not loading. 
Has anyone else faced this issue?

**Comment by anthonycaccese** [comment]
can i ask for details on the set up you have tried?

**Comment by trashcatt** [comment]
I've added the images to the system screenshot folder and added the SVGs to the logos folder. The backgrounds work fine but the SVGs don't show. I tested it by renaming one of the existing logos to the name of the custom system (ex. gbaph for gba pokemon hacks) and those work fine. I'm assuming it has something to do with Photoshop since that's what I'm using. I'm not 100% sure that's the case but it seems to be having some effect.

**Comment by anthonycaccese** [comment]
its possibly related to the name of the images compared to the name of your collection.  Can you please add those details here (along with the images)?

**Comment by trashcatt** [comment]
![Screenshot 2024-12-31 153154](https://github.com/user-attachments/assets/97420dbe-d25c-4663-9fda-f82a4c282f53)
**es_systems.xml**
![Screenshot 2024-12-31 153446](https://github.com/user-attachments/assets/585c468c-7ce9-4e9b-af46-686de9e5aef7)
![Screenshot 2024-12-31 153505](https://github.com/user-attachments/assets/61e29405-ce5a-4dbe-8092-16f5e53a6935)

**Comment by anthonycaccese** [comment]
hmm, I am not sure what is happening based on the details you provided.  Testing on my end I was able to add a custom system and then add the logo and artwork files to the corresponding folder in the theme and the images show as expected.

Please make sure you are placing your images into the right folders based on the color scheme you are using.   And you will need 2 images for each system - the logo (which should be an svg) and the diagonal artwork (which should be a png)

**Comment by trashcatt** [comment]
Yeah, it's weird. It seems like everything should be working but the logos aren't. Here's a screenshot with the working background image. ![Screenshot_20250104-141811](https://github.com/user-attachments/assets/f6cf7e62-f336-4420-9f73-13569006c985)

Just to make sure things are in the correct folders. For this example I have a custom folder named `gbaph` and I have the logos in `ES-DE/themes/art-book-next-es-de/_inc/systems/logos/gbaph.svg`  

I feel like it has to be something with my svgs.

**Comment by anthonycaccese** [comment]
yeah that is the right folder.  the svg might be it.   can you attach your svg for gbaph here for me to take a look at?

**Comment by trashcatt** [comment]
![gbaph](https://github.com/user-attachments/assets/61512f00-da85-4b30-b126-377c0ca313a7)

**Comment by anthonycaccese** [comment]
thank you!  Yes its the SVG, the theme engine does not support displaying embedded images inside SVGs so you would need to convert the image to a set of paths instead in order for it to display correctly.  

Sadly thats not something I can do from the theme end at all as its an engine detail.  

That said, paths are preferred because of their ability to scale to any resolution and retain clarity so thats why I am using SVGs for the logos to begin with.

**Comment by trashcatt** [comment]
That was it! This was my first time really creating my own SVGs. I just had to make it in Illustrator with paths instead and everything works as expected. Thank you so much for your help!

![Screenshot_20250106-220637](https://github.com/user-attachments/assets/0c8a4689-7786-4c7e-b7be-76e6369e8a85)

**Comment by anthonycaccese** [comment]
Thank you for bringing it up and glad to help.
